data-prepper-plugins/mongodb/src/main/java/org/opensearch/dataprepper/plugins/mongo/stream/StreamWorker.java

```diff
@@ -221,6 +243,7 @@ record = document.getFullDocument().toJson(writerSettings);
                             }
                         } catch(Exception e){
                             // TODO handle documents with size > 10 MB.
+                            // collection.find(new Document("_id", "key")).first();
```

Comment:
Did you want to keep this?

Reply:
I plan to implement that todo later. Will remove this for now.